### PR TITLE
fix(countbadge): add horizontal padding to count

### PR DESCRIPTION
**Pull request by alexkrolick**
Screen Shot 2019-03-14 at 4 45 40 PM
Screen Shot 2019-03-14 at 4 45 22 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

**Comment by booc0mtaco** [comment]
Can you include a screenshot of the `badgeable` example with this change?

**Comment by alexkrolick** [comment]
@booc0mtaco see attached

File: src/components/count-badge/CountBadge.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
     height: 16px;
     line-height: 16px;
     min-width: 16px;
-    padding: 0 2px;
+    padding: 0 4px;
```
Comment (booc0mtaco) [inline src/components/count-badge/CountBadge.scss]: we'll soon start coalescing on a 4-pixel grid for our sizings, so this component is now more aligned on that

**Pull request merged**